plugins/woocommerce-admin/client/subscriptions/index.tsx

```diff
@@ -176,7 +208,10 @@ const MainContent: React.FC< MainContentProps > = ( { setHasError } ) => {
 				<TermsOfService />
 			</p>
 
-			<GetStartedButton setHasError={ setHasError } />
+			<div className="wcpay-empty-subscriptions__button_container">
+				<NoThanksButton></NoThanksButton>
+				<GetStartedButton setHasError={ setHasError } />
```

Comment:
We might need a design person to confirm this but my gut feeling is that the "Get started" button should appear before the "No thanks" button, i.e. switch around the order of the buttons.

Reply:
I honestly thought the same thing but followed the same order used in the [other experiment page](https://user-images.githubusercontent.com/8490476/167322768-82e51783-6c54-4431-acd1-19c97ae2883d.png). 

I'm happy to reorder it to put the primary button first. Changed in [59478e3](https://github.com/woocommerce/woocommerce/pull/32913/commits/59478e391c51cdc38adf12cae80988a019126c40)